internal/integrations/envsec/envsec.go

```diff
@@ -119,15 +120,14 @@ func handleError(stderr *bytes.Buffer, err error) error {
 	return errors.WithStack(err)
 }
 
-func DefaultEnvsec(stderr io.Writer, workingDir string) *envsec.Envsec {
+func DefaultEnvsec(stderr io.Writer) *envsec.Envsec {
```

Comment:
Given that most uses of this function are setting `WorkingDir`, I feel we should preserve the parameter. Otherwise, its very easy to overlook or forget it needs to be set.

Can we pass in an empty-value for the "WhoAmI" use-case?